frappe/desk/doctype/system_console/system_console.js

```diff
@@ -30,6 +21,17 @@ frappe.ui.form.on("System Console", {
 				.then(() => frm.trigger("render_sql_output"))
 				.finally(() => $btn.text(__("Execute")));
 		});
+		if (
+			window.localStorage.getItem("system_console_code") &&
+			window.localStorage.getItem("system_console_type")
+		) {
+			frm.set_value("type", localStorage.getItem("system_console_type"));
+			frm.set_value("console", localStorage.getItem("system_console_code"));
+			frm.set_value("output", "");
+			window.localStorage.removeItem("system_console_code");
+			window.localStorage.removeItem("system_console_type");
+			$(".primary-action[data-label='Execute']").click();
```

Comment:
```suggestion
```

Do not automatically execute. Kinda dangerous.

Reply:
Added this so that as soon as we click the button it is executed in System Console(Hussain wanted me to add this)

Reply:
This is dangerous TBH. One extra click doesn't hurt here.